src/store/cockpitApi.ts

```diff
@@ -89,8 +91,34 @@ export const cockpitApi = emptyCockpitApi.injectEndpoints({
           }
         },
       }),
+      deleteBlueprint: builder.mutation<
+        DeleteBlueprintApiResponse,
+        DeleteBlueprintApiArg
+      >({
+        // invalidatesTags: [{ type: 'Blueprints' }],
+        queryFn: async ({ id: filename }) => {
+          try {
+            const path = await getBlueprintsPath();
+            const filepath = path + '/' + filename;
+
+            await cockpit.spawn(['rm', '-r', filepath], {
```

Comment:
Does the `-r` help here?
If it's a file, it will be deleted without the argument, too. If it's a non-empty directory, the call will fail.

Reply:
Think I meant to put an -f flag rather than the -r. But maybe we should leave that off too